core/common/src/main/java/alluxio/wire/MasterWebUIInit.java

```diff
@@ -117,7 +117,7 @@ public MasterWebUIInit setDebug(boolean debug) {
    * Sets newer version available.
    *
    * @param newerVersionAvailable the newer version available
-   * @return true if a newer version is available, false otherwise
+   * @return the newer version available
```

Comment:
wait this is still wrong

Reply:
actually all `set*` methods have an incorrect javadoc in this file

Reply:
i was trying to match the other `set` method `@return` comments. what would be better?

Reply:
you could fix all methods. with `@return the updated MasterWebUIInit instance`

Reply:
updated